packages/paste-design-tokens/types/index.ds.ts

```diff
@@ -1,2 +1,2 @@
 declare module 'gulp-theo';
-declare module 'gulp-restart';
+export {};
```

Comment:
What does this do? 🤔

Reply:
It's based on a TSconfig rule we have for all the other packages. I think for this package it's safe to turn off isolatedModules, so I can get rid of the empty export and stop typescript complaining.